scripts/copy-wellknown.cjs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+
+const wellKnownPath = path.join(__dirname, '../.well-known');
+
+console.log('Copying .well-known/ contents to apps');
```

Comment:
is this console log necessary?

Reply:
It'll double log since we also echo in the pre-commit but I'd say this is also fine 😄